crates/rust-analyzer/Cargo.toml

```diff
@@ -28,6 +28,7 @@ rustc-hash = "1.1.0"
 serde = { version = "1.0.106", features = ["derive"] }
 serde_json = "1.0.48"
 threadpool = "1.7.1"
+rayon = "1.3.1"
```

Comment:
Just seeing rayon in Cargo.toml creates an [impression of speed](https://open.spotify.com/album/2qpUTpadrGai80AMFLyqSq?highlight=spotify:track:1dFzF1MmwSl9HTh1j7W5HG) :+1: !